src/libnnstreamer-edge/nnstreamer-edge-util.c

```diff
@@ -170,7 +169,7 @@ nns_edge_malloc (nns_size_t size)
     mem = malloc (size);
 
   if (!mem)
-    nns_edge_loge ("Failed to allocate memory (%llu).", size);
+    nns_edge_loge ("Failed to allocate memory");
```

Comment:
Is there any reason to change this line?

Reply:
%lu or %llu is required when building in 32bit and 64bit with gbs.
I tried to solve it with glib API, but nnstreamer-edge does not have glib dependency.
There is also a way to use the C library.
Since there was only one log, I agreed with @gichan-jang to delete it for now.

Reply:
A major NNStreamer VoC is that error messages do not give you enough info. Please do not reduce info in error messages. Please add more info.

Reply:
Okay, thank you.